contracts/AedexV2Factory.aes

```diff
@@ -42,7 +42,9 @@ contract AedexV2Factory =
 
     entrypoint all_pairs_length(): int = List.length(state.all_pairs)
 
-    entrypoint all_pairs(ix: int): IAedexV2Pair = List.get(ix, state.all_pairs)
+    entrypoint get_all_pairs(): list(IAedexV2Pair) = state.all_pairs
+
+    entrypoint get_nth_pair(ix: int): IAedexV2Pair = List.get(ix, state.all_pairs)
```

Comment:
Why did you prefer to use `List.get` over `List.nth` in `get_nth_pair`? I noticed that the previously existing `get_pair` returns `option(IAedexV2Pair)` which is more difficult to be used in a wrong way (when given an out of bounds index).

Reply:
- the `get_pair` entrypoint is used to get the instance of a pair based on some arbitrary 2 tokens, but also at the same time to see if that pair exists. So , `option(IAedexV2Pair)` serves both purposes
- `get_nth_pair` is used in the context of `all_pairs_length`, from which the caller should know in advance the length of the whole stored pairs. So , the failure in the case of an out of boundary index is an assumed one.